src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/HeaderCell.js

```diff
@@ -1,17 +1,77 @@
 // @flow
 import type {Node} from 'react';
 import React from 'react';
+import classNames from 'classnames';
+import Icon from '../Icon';
 import tableStyles from './table.scss';
 
+const ASCENDING_ICON = 'chevron-up';
+const DESCENDING_ICON = 'chevron-down';
+
 type Props = {
     children?: Node,
+    /**
+     * @ignore 
+     * If set to true, the cell appears as a header-control-cell
+     */
+    isControl?: boolean,
+    /** Called when column was clicked */
+    onClick?: () => void,
+    /** If set, an indicator will show up */
+    sortMode: 'default' | 'ascending' | 'descending',
```

Comment:
Wouldn't `none` be more appropriate than `default`? It's not sorted at all by this column in most cases.